pkgs/applications/kde/granatier.nix

```diff
@@ -0,0 +1,17 @@
+{ mkDerivation, lib
+, libkdegames, extra-cmake-modules
+, kdeclarative, knewstuff
+}:
+
+mkDerivation {
+  name = "granatier";
```

Comment:
This seems to match the pattern set by other KDE applications, but I would have expected to see `version` defined somewhere. But I'm not that familiar with the KDE infrastructure.

Reply:
true, there is no version attribute in the derivation, but the name of the generated package still has one, I didn't look how they did it,

don't know how to properly fix this

Reply:
It is possibly some KDE infrastructure magic, and you seem to be following common practice for that section of nixpkgs.

Reply:
The version is set for all `kdeApplications` at once, because they are upgraded as a unit.